### DISCUSSION TITLE
Can we recommend new bad-user agent?

### DISCUSSION THREAD

**Discussion by glikis** [mitchellkrogza/nginx-ultimate-bad-bot-blocker#578] [General]
Hello,

We noticed that the following bot ClaudeBot is really aggressive and it is not included in your list.

Thank you

**Comment by mitchellkrogza** [comment]
Yes of course you can use the Issue template for this

https://github.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker/issues/new?assignees=mitchellkrogza&labels=Bots+%2F+User-Agents&projects=&template=report-user-agent-bad-bot.md&title=%5BUser-Agent%5D+%28add+a+descriptive+title+here%29
